kolibri/core/assets/src/views/k-linear-loader.vue

```diff
@@ -14,26 +14,39 @@
 
   import uiProgressLinear from 'keen-ui/src/UiProgressLinear';
 
+  /**
+   * Used to show determinate or indeterminate loading
+   */
   export default {
     name: 'kLinearLoader',
     components: {
       uiProgressLinear,
     },
     props: {
+      /**
+       * Whether there should be a delay before the loader displays
+       */
       delay: {
         type: Boolean,
         required: true,
       },
+      /**
+       * Determinate or indeterminate
+       */
```

Comment:
this is fine, but curious why you went with a string instead of a boolean?

Reply:
I guess just a bit more explicit.